.github/workflows/op-succinct-docker-image-builder.yml

```diff
@@ -0,0 +1,61 @@
+---
+  # Workflow for automatically building and deploying docker images for kurtosis-cdk.
+  name: OP-Succinct Docker Image Builder
+  
+  on:
+    workflow_dispatch:
+      inputs:
+        agglayer-op-succinct-commit:
+          description: The agglayer/op-succinct branch or commit to build the image from
+          required: true
+  
+  env:
+    IMAGE_NAME: leovct/op-succinct-contract-deployer
+    RUST_VERSION: 1.84.1
+    FOUNDRY_VERSION: nightly
+    OP_SUCCINCT_BRANCH: ${{ github.event.inputs.agglayer-op-succinct-commit }}
```

Comment:
not sure if this line would work? I guess you could remove this env variable and use `${{ github.event.inputs.agglayer-op-succinct-commit }}` directly in the step